### DISCUSSION TITLE
Postgres - Migrate issue

### DISCUSSION THREAD

**Discussion by gaurishhs** [prisma/prisma#15299] [Q&A]
Hello people, I'm having this issue (using prisma for the first time)

My model is as such:
```
model Reminder {
  userId String 
  id String @id @default(uuid()) @unique
  content String
  endsAt DateTime
}
```

Error:
:warning: We found changes that cannot be executed:

  • Step 0 The required column id was added to the Reminder table with a prisma-level default value. There are 1 rows in this table, it is not possible to execute this step. Please add this column as optional, then populate it before making it required.

You can use prisma migrate dev --create-only to create the migration file, and manually modify it to address the underlying issue(s).
Then run prisma migrate dev to apply it and verify it works.

Upon running create only and then migrate it says:
```
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20220913175756_init

Database error code: 23502

Database error:
ERROR: column "id" of relation "Reminder" contains null values

DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E23502), message: "column \"id\" of relation \"Reminder\" contains null values", detail: None, hint: None, position: None, where_: None, schema: Some("public"), table: Some("Reminder"), column: Some("id"), datatype: None, constraint: None, file: Some("tablecmds.c"), line: Some(5443), routine: Some("ATRewriteTable") }
```

**Comment by janpio** [comment]
Did you modify the migration in a way that addresses the underlying issues? (In this case, running a query that removed the null values in that column for example could work.)

**Reply by vladi-stevanovic** [reply]
👋 Hello @gaurishhs did you perhaps have a chance to check Jan's recommendation? Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
